org.eclipse.jgit.test/tst/org/eclipse/jgit/lib/ConfigTest.java

```diff
@@ -269,13 +271,22 @@ public void test007_readUserConfig() {
 
 	@Test
 	public void testReadUserConfigWithInvalidCharactersStripped() {
+		String authorName = null;
+		try {
+			authorName = SystemReader.getInstance().getenv(Constants.GIT_AUTHOR_NAME_KEY);
+		} catch (AccessControlException e) {
+			// do nothing;
```

Comment:
Why are we doing nothing here?

Reply:
If the env variable GIT_AUTHOR_NAME_KEY is not accessible, authorName shall be set to null, therefore nothing to be done here.